src/middlewares/roleMiddlewares.js

```diff
@@ -0,0 +1,17 @@
+import {Users} from "../database/models";
+import {AuthorizationError, internalServerError} from "../utils/errors/applicationsErrors"
+
+const  isRequester = async (req, res, next) => {
```

Comment:
Rebase and this login in authorize middleware to avoid having many files with the same role and remove this